perf/client/bench/components/victory-area.bench.js

```diff
@@ -0,0 +1,388 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import VictoryArea from "src/components/victory-area/victory-area";
+import VictoryChart from "src/components/victory-chart/victory-chart";
+import VictoryGroup from "src/components/victory-group/victory-group";
+import VictoryStack from "src/components/victory-stack/victory-stack";
+import { element, suiteOpts } from "../bench-helpers.js";
+
+suite("<VictoryArea> data", () => {
+  benchmark("identity fn", () => ReactDOM.render(<VictoryArea />, element));
+
+  benchmark("10 pts", () => ReactDOM.render(
+    <VictoryArea
+      data={[
+        {x: 1, y: 1},
```

Comment:
Please extract all these data arrays into a separate file, or better yet make a couple functions to generate random data sets of a given size